dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -375,6 +375,10 @@ public void manualPublish(final List<String> args) {
             final String privateAccess = optVal(args, "--private", "false");
             final String customDockerPath = optVal(args, "--custom-docker-path", null);
 
+            if (toolname != null && toolname.startsWith("_")) {
```

Comment:
Better to enforce this at a webservice and database constraint level rather than in the client (if it has not been already). That would cover the UI, anyone using the API directly, and, well, us (accidentally).

Reply:
Would the db constraint be something like can only start with _ if parentEntry is not null (implies it is a checker)?

Reply:
Yeah, or a bit more broad:
A checker workflow must start with "_", a non-checker workflow cannot start with "_" (if the name is non-null)